flink-docs/src/main/java/org/apache/flink/docs/configuration/ConfigOptionsDocGenerator.java

```diff
@@ -260,10 +262,26 @@ private static String toHtmlTable(final List<OptionWithMetaInfo> options) {
 	private static String toHtmlString(final OptionWithMetaInfo optionWithMetaInfo) {
 		ConfigOption<?> option = optionWithMetaInfo.option;
 		String defaultValue = stringifyDefault(optionWithMetaInfo);
+		Documentation.TableMeta tableMeta = optionWithMetaInfo.field.getAnnotation(Documentation.TableMeta.class);
+		StringBuilder sb = new StringBuilder();
+		if (tableMeta != null) {
+			Documentation.ExecMode execMode = tableMeta.execMode();
+			if (Documentation.ExecMode.BATCH_STREAMING.equals(execMode)) {
+				sb.append("<br> <span class=\"label label-primary\">")
+						.append("BATCH")
```

Comment:
Use `ExecMode.BATCH` instead.

Reply:
I think BATCH is enough.

Reply:
I believe the point was to use `ExecMode.BATCH.toString()`, to ensure it is in sync with the `else` branch.

Reply:
Yes.

Reply:
Hi @XuPingyong , could you update this too for a better protection?